ftw/tokenauth/service_keys/browser/view_usage_logs.pt

```diff
@@ -34,13 +34,15 @@
                 <thead>
                     <tr>
                         <th i18n:translate="">Time</th>
+                        <th i18n:translate="label_user_id">User ID</th>
```

Comment:
Could you update the German translations?

Reply:
`label_user_id` is already there. Or do you mean the commented lines that list the occurrences? The problem with those is that they would need to get updated on particularly every commit because of changing line numbers. So far I'm only updating message files if there are changes in the translations.

Reply:
ah, sorry, I agree; I assumed that the label is new.